cypress/e2e/tests/pages/manager/kontainer-drivers.spec.ts

```diff
@@ -231,7 +231,6 @@ describe('Kontainer Drivers', { testIsolation: 'off', tags: ['@manager', '@admin
         cy.wait('@deleteOracleDriver').its('response.statusCode').should('eq', 200);
 
         driversPage.waitForPage();
-        driversPage.list().details(linodeDriver, 1).contains('Removing');
```

Comment:
I only saw this status update once in 2.9-head. In general it seems like the resources are deleted before the status is updated. There's still validation to ensure that the drivers are removed from the rows so I don't think it's necessary to keep this.